### PR TITLE
Fix read of FPC register

### PR DESCRIPTION
**Pull request by keithc-ca**
Update JIT to match https://github.com/eclipse/omr/pull/3509.

### CODE REVIEW

File: runtime/compiler/runtime/SignalHandler.c [hunk 1]
```diff
@@ -920,7 +920,7 @@ UDATA jit390Handler(J9VMThread* vmThread, U_32 sigType, void* sigInfo)
       const char* infoName;
       UDATA *controlPC;
       UDATA controlPCValue;
-      UDATA *controlFPC;
+      U_32 *controlFPC;
```
Comment (fjeremic) [inline runtime/compiler/runtime/SignalHandler.c]: While we're in this area of code we can remove this variable all together. All we care about is the value of the `DXC` on line 948. We can fold `infoValue` directly into it and remove the `controlFPC` variable.

Comment (keithc-ca) [reply runtime/compiler/runtime/SignalHandler.c]: Sure, will do.

Comment (keithc-ca) [reply runtime/compiler/runtime/SignalHandler.c]: Done.

**Comment by fjeremic** [comment]
Jenkins test sanity zlinux JDK8 depends eclipse/omr#3509

**Comment by fjeremic** [comment]
Relaunching the build because of issues introduced in #4352.

Jenkins test sanity zlinux JDK8 depends eclipse/omr#3509

**Comment by fjeremic** [comment]
Relaunching the build because of issues introduced in #4352.

Jenkins test sanity zlinux JDK8 depends eclipse/omr#3509

**Pull request merged**